src/test/autodjprocessor_test.cpp

```diff
@@ -77,6 +77,10 @@ class FakeDeck : public BaseTrackPlayer {
         play.set(bPlay);
     }
 
+    void slotCopyFrom(const QString& group) override {
+        return;
+    }
```

Comment:
I'm not sure what should go here.

Reply:
[MOCK_METHOD1](https://github.com/google/googletest/blob/master/googlemock/docs/ForDummies.md#writing-the-mock-class)?